### PR TITLE
Fix allowing ports while starting Orion issue#3875

### PR DESCRIPTION
**Pull request by ArqamFarooqui110719**
Fix allowing ports while starting Orion issue #3875

### CODE REVIEW

**Comment by ArqamFarooqui110719** [comment]
Hi @fgalan Sir,
Kindly review this PR. 
If it is OK then please merge the PR.

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
 - Add: -mqttTimeout (env var ORION_MQTT_TIMEOUT) for MQTT broker connection timeout
 - Fix: queue workers deadlock when MQTT broker was not responding to connection attemp
 - Fix: Ngsiv2-AttrsFormat header set to "normalized" in NGSI patching custom notification
+- Fix: Orion should return error and refuse to start when port is negative
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:4]: Just a small suggestion to include the related issue number, pls:

```suggestion
- Fix: Orion should return error and refuse to start when port is negative (3875)
```

Comment (ArqamFarooqui110719) [reply CHANGES_NEXT_RELEASE]: Fixed in [5cd0e52](https://github.com/telefonicaid/fiware-orion/pull/4287/commits/5cd0e52824c063e8b0d7638ea875ab2bf4e15341)

**Review by fgalan — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**